### PR TITLE
changes repos links to synerbi

### PR DESCRIPTION
**Pull request by paskino**

### CODE REVIEW

File: scripts/UPDATE.sh [hunk 1]
```diff
@@ -120,11 +120,14 @@ SuperBuild(){
...
   then
-    git clone https://github.com/CCPPETMR/SIRF-SuperBuild.git
+    git clone https://github.com/SynerBi/SIRF-SuperBuild.git
     cd SIRF-SuperBuild
   else
     cd SIRF-SuperBuild
+    # update the remote repository after rename to SynerBI
+    git remote set-url origin https://github.com/SynerBI/SIRF-SuperBuild.git
     git fetch
+    git pull
```
Comment (KrisThielemans) [inline scripts/UPDATE.sh:130]: can't do an explicit `pull`. The rest of the script will take care of it. `fetch` updates the content of the remote branches. `pull` would do `fetch` and `merge`

Comment (KrisThielemans) [inline scripts/UPDATE.sh:130]:
```suggestion
```

**Comment by KrisThielemans** [comment]
we probably need to manage that an old VM will have files in `~/devel/CCPPETMR_VM` which will now have been changed.

This most likely also affects this wiki

File: scripts/UPDATE.sh (hunk 1 above)
Comment (paskino) [reply scripts/UPDATE.sh:130]: Sure? I added it just because fetch wouldn't suffice. I'll try again.

Comment (paskino) [reply scripts/UPDATE.sh:130]: It seems to me that we only request:
1. fetch
1. checkout to a different tag

I don't see how the rest of the script does a pull of the `SyneRBI_VM` if we don't change branch. Am I wrong?

Comment (KrisThielemans) [reply scripts/UPDATE.sh:130]: I have thought more about this. I've created #153. Let's not fix this in this PR though.

Note that a  `pull` *before* the checkout doesn't change anything in the first 2 cases mentioned in #153, and potentially updates the wrong local branch.

Comment (paskino) [reply scripts/UPDATE.sh]: I removed the `pull`

**Comment by paskino** [comment]
What about creating a symbolic link? `SyneRBI_VM -> CCPPETMR_VM`

File: scripts/UPDATE.sh [hunk 2]
```diff
@@ -120,10 +120,12 @@ SuperBuild(){
   cd $SIRF_SRC_PATH
   if [ ! -d SIRF-SuperBuild ] 
   then
-    git clone https://github.com/CCPPETMR/SIRF-SuperBuild.git
+    git clone https://github.com/SynerBi/SIRF-SuperBuild.git
     cd SIRF-SuperBuild
   else
     cd SIRF-SuperBuild
+    # update the remote repository after rename to SynerBI
+    git remote set-url origin https://github.com/SynerBI/SIRF-SuperBuild.git
```
Comment (KrisThielemans) [inline scripts/UPDATE.sh:128]:
```suggestion
    git remote set-url origin https://github.com/SyneRBI/SIRF-SuperBuild.git
```
and all others

**Comment by KrisThielemans** [comment]
Needs repo changes in `bootstrap_ubuntu.sh` and possibly other files

**Comment by KrisThielemans** [comment]
> What about creating a symbolic link? SyneRBI_VM -> CCPPETMR_VM

good idea, if CCPPETMR_VM  exists and SyneRBI_VM doesn't, create the link

**Comment by KrisThielemans** [comment]
`update.sh` needs to use `~/devel/SyneRBI_VM`. Probably other things as well.

**Comment by paskino** [comment]
> > What about creating a symbolic link? SyneRBI_VM -> CCPPETMR_VM
> 
> good idea, if CCPPETMR_VM exists and SyneRBI_VM doesn't, create the link

I'm thinking that this is going to complicate the things in a consistent way for no real reason.  The important bit is to update the github links (though not fundamental either). 

Wouldn't it be better to create a new VM? We may then release it with SIRF as 2.1.x.

**Comment by KrisThielemans** [comment]
> I'm thinking that this is going to complicate the things in a consistent way for no real reason.

You mean creating the `CCPPETMR_VM` link? Or change `update.sh`? Things have to work, both for a new one and for updating an existing (but not too old) one. Not everyone is using their VMs for a short time (otherwise, why do we have the update mechanism at all?).

> Wouldn't it be better to create a new VM? We may then release it with SIRF as 2.1.x.

Agreed that it would be best to create a new VM, but it seems hard to do it without a new release. how are we going to call it otherwise?

**Comment by paskino** [comment]
created a VM from scratch with this branch and it works. I'll try to upload it somewhere for test.

**Comment by paskino** [comment]
created a VM from scratch with this branch and it works. I'll try to upload it somewhere for test.

**Comment by KrisThielemans** [comment]
This looks ok to me now. However, I did spot a problem with the automatic `set remote`. On my VM, I had changed the remote manually to something else (actually `git@github.com:...` such that I don't need to type passwords all the time). Doing on `update_VM.sh` got rid of all these.

As the ccppetmr redirects are all working fine, I suggest that we remove these. People with old VMs will have no problems. People with new VMs will get the synerbi repo.

Objections? @casperdcl @paskino

**Comment by KrisThielemans** [comment]
I've introduced an option for the remote. Also changed the "version" stuff a bit.

**Pull request merged**